### PR TITLE
Add Storage and Cookies doc to What's New

### PR DESCRIPTION
**Pull request by oliverdunk**
Adds the new [Storage and Cookies](https://developer.chrome.com/docs/extensions/mv3/storage-and-cookies/) documentation to [What's New](https://developer.chrome.com/docs/extensions/whatsnew/).

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Documentation on cookies and web storage APIs {: #storage-and-cookies-guide }
+
+<p class="color-secondary-text type--caption">Posted on <time>October 2, 2023</time></p>
+
+We published [a new guide](/docs/extensions/mv3/storage-and-cookies/) on how cookies and web storage
+APIs work in Chrome extensions. It includes all you need to know about
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:23]:
```suggestion
We published a new guide on [how cookies and web storage APIs work](/docs/extensions/mv3/storage-and-cookies/) in Chrome extensions. It includes all you need to know about
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: 'layouts/doc-post.njk'
 title: What's new in Chrome extensions
 description: 'Recent changes to the Chrome extensions platform, documentation, and policy'
 date: 2021-02-25
-updated: 2023-09-29
+updated: 2023-10-02
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:6]:
```suggestion
updated: 2023-10-05
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 3]
```diff
@@ -15,6 +15,14 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Documentation on cookies and web storage APIs {: #storage-and-cookies-guide }
+
+<p class="color-secondary-text type--caption">Posted on <time>October 2, 2023</time></p>
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:20]:
```suggestion
<p class="color-secondary-text type--caption">Posted on <time>October 5, 2023</time></p>
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — CHANGES_REQUESTED** [review]
We should not assume that extension developers know what privacy sandbox is. Can you please add a _brief_ explanation of what it is and why it's important.

File: site/en/docs/extensions/whatsnew/index.md [hunk 4]
```diff
@@ -19,8 +19,10 @@ Check this page often to learn about changes to Chrome extensions, extensions do
 
 <p class="color-secondary-text type--caption">Posted on <time>October 9, 2023</time></p>
 
-We published a new guide on [how cookies and web storage APIs work](/docs/extensions/mv3/storage-and-cookies/) in Chrome extensions. It includes all you need to know about
-[Privacy Sandbox](/docs/privacy-sandbox/) as an extension developer.
+We published a new guide on [how cookies and web storage APIs work in Chrome extensions](/docs/extensions/mv3/storage-and-cookies/).
+It includes details on how cookie and storage partitioning changes in
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:23]:
```suggestion
It includes details on cookie and storage partitioning changes in
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 5]
```diff
@@ -19,8 +19,10 @@ Check this page often to learn about changes to Chrome extensions, extensions do
...
+[Privacy Sandbox](/docs/privacy-sandbox/), an ongoing project to deprecate third-party cookies
+through the creation of a series of new APIs, work in extensions.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:25]:
```suggestion
through the creation of a series of new web platform APIs, and work in extensions.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md (hunk 5 above)
Comment (oliverdunk) [reply site/en/docs/extensions/whatsnew/index.md:25]: Accepting this but removed the "and" (it's not that work is being done in extensions, it's that we're telling you how something works/behaves).

File: site/en/docs/extensions/whatsnew/index.md [hunk 6]
```diff
@@ -15,6 +15,15 @@ tags:
...
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Documentation on cookies and web storage APIs {: #storage-and-cookies-guide }
+
+<p class="color-secondary-text type--caption">Posted on <time>October 9, 2023</time></p>
+
+We published a new guide on [how cookies and web storage APIs work in Chrome extensions](/docs/extensions/mv3/storage-and-cookies/).
+It includes details on cookie and storage partitioning changes in
+[Privacy Sandbox](/docs/privacy-sandbox/), an ongoing project to deprecate third-party cookies
+through the creation of a series of new web platform APIs, work in extensions.
```
Comment (jpmedley) [inline site/en/docs/extensions/whatsnew/index.md:25]:
```suggestion
It includes details on cookie and storage partitioning changes in
[Privacy Sandbox](/docs/privacy-sandbox/), an ongoing project to deprecate third-party cookies
through the creation of a series of new web platform APIs, and on how they work in extensions.
```

**Review by jpmedley — APPROVED** [review]

**Pull request merged**